tsconfig.json

```diff
@@ -0,0 +1,13 @@
+{
+  "compilerOptions": {
+    "target": "ES2020",
+    "composite": true,
```

Comment:
I've had a look at some of my typescript monorepos, this is not required. Do you have a functional reason for including this that affects the project? Generally when you stray from the `tsconfig.json` defaults it causes more issues...